src/dashboard/src/templates/archival_storage/view.html

```diff
@@ -0,0 +1,182 @@
+    {% breadcrumb_url 'Archival storage' 'archival_storage_index' %}
+    {% breadcrumb name %}
+  </ul>
+
+  <h2 class="heading-aip-details">
+    {{ name }}
+    <small>{% trans "Archival Information Package" %}</small>
+  </h2>
+
+  <table class="bordered-table aip-details">
+    <tbody>
+      <tr>
+        <th>UUID</th>
+        <td><code class="uuid">{{ uuid }}</code></td>
+      </tr>
+      <tr>
+        <th>{% trans "Size" %}</th>
+        <td>{{ size }}</td>
+      </tr>
+      <tr>
+        <th>{% trans "Date stored" %}</th>
+        <td><span class="timestamp">{{ created }}</span></td>
+      </tr>
+      <tr>
+        <th>{% trans "Status" %}</th>
+        <td>{{ status }}</td>
+      </tr>
+      <tr>
+        <th>{% trans "Location" %}</th>
+        <td>
+          <a class="btn small" href="{% url 'components.archival_storage.views.aip_download' uuid %}">{% trans "Download" %}</a>
+          <code class="location-preview">[...]/{{ location_basename }}</code>
+        </td>
+      </tr>
+      <tr>
+        <th>{% trans "Pointer file" %}</th>
+        <td>
+          <a class="btn small" target="_blank" href="{% url 'components.archival_storage.views.aip_pointer_file_download' uuid %}">{% trans "View" %}</a>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+
+  <h3>{% trans "Actions" %}</h3>
+
+  <section class="am-tabs-pane">
+
+    <ul class="tabs" data-active-tab="{{ active_tab }}">
+      <li class="active"><a href="#tab-upload-dip">{% trans "Upload DIP" %}</a></li>
+      <li><a href="#tab-reingest">{% trans "Re-ingest" %}</a></li>
+      <li><a href="#tab-delete">{% trans "Delete" %}</a></li>
+    </ul>
+
+    <div class="tab-content">
+
+      <!-- Upload tab -->
+      <div class="tab-pane active" id="tab-upload-dip">
+        <form action="" method="POST" class="form-stacked">
+          {% csrf_token %}
+          <fieldset>
+            <legend>{% trans "Metadata-only DIP upload to AtoM" %}</legend>
+            <div class="clearfix">
+              {{ forms.upload.slug.label_tag }}
+              <div class="input">
+                {{ forms.upload.slug.errors }}
+                {{ forms.upload.slug }}
+              </div>
+            </div>
+          </fieldset>
+          <div class="clearfix am-actions">
+            <button type="submit" class="btn primary" name="submit-upload-form">{% trans "Upload" %}</button>
+          </div>
+        </form>
+      </div>
+
+      <!-- Re-ingest tab -->
+      <div class="tab-pane" id="tab-reingest">
+        <form action="" method="POST" class="form-stacked">
+          {% csrf_token %}
+          <fieldset>
+            <legend>{% trans "Re-ingest" %}</legend>
+            <div class="clearfix">
+              <label>{% trans "Choose type of re-ingest:" %}</label>
+              <div class="input">
+                {{ forms.reingest.reingest_type.errors }}
+                <ul class="inputs-list">
+                  {% for choice in forms.reingest.reingest_type %}
```

Comment:
Any reason the forms aren't done with Django's built-in form display, or Archivematica's form display (`{% include "_form.html" with form=forms.reingest %}`) ? It would be more flexible with new changes, such as full reingest's extra fields.